scripts/examples/index.csv

```diff
@@ -27,6 +27,7 @@
 "examples/09-WiFi/WINC1500", "(OPENMV2|OPENMV3|OPENMV4|OPENMV4P|OPENMVPT)", ".+", ""
 "examples/10-Bluetooth", "(OPENMV_RT1060|ARDUINO_GIGA|ARDUINO_PORTENTA_H7|ARDUINO_NICLA_VISION|ARDUINO_NANO_RP2040_CONNECT)", ".+", ""
 "examples/50-OpenMV-Boards/(?!main\.py).*$", "(OPENMV.*)", ".+", "OpenMV-Boards$"
+"examples/50-OpenMV-Boards/62-Pure-Thermal", "(OPENMVPT)", ".+", "OpenMV-Boards$"
```

Comment:
The pure thermal example is added to the menu for all OpenMV boards (because the rule above matches everything), this rule doesn't really do anything useful. I think we'll need to reorder the examples anyway to support board-control examples specific to imxrt.

Reply:
Doesn't the "(OPENMVPT)" only match the pure thermal?